NBitcoin/BitcoinStream.cs

```diff
@@ -265,6 +265,25 @@ public void ReadWrite<T>(ref List<T> list) where T : IBitcoinSerializable, new()
 			}
 		}
 
+		public  void ReadWriteListBytes<TList>(ref TList data)
+			where TList : List<byte[]>, new()

```

Comment:
Why making it generic? there is no reason to.

Reply:
done